src/java.desktop/share/classes/sun/font/FontManagerFactory.java

```diff
@@ -25,72 +25,31 @@
 
 package sun.font;
 
-import java.awt.AWTError;
-import java.awt.Font;
-import java.awt.GraphicsEnvironment;
-import java.awt.Toolkit;
-import java.security.AccessController;
-import java.security.PrivilegedAction;
-
-import sun.security.action.GetPropertyAction;
-
-
 /**
  * Factory class used to retrieve a valid FontManager instance for the current
  * platform.
  *
  * A default implementation is given for Linux, Solaris and Windows.
```

Comment:
Should we also remove mention of Solaris from here (due to JEP381) not being supported in mainline?

Reply:
I updated the comment to mention Mac OS instead of Solaris: `A default implementation is given for Linux, Mac OS and Windows.`